docs/references/general/rawacf.md

```diff
@@ -1,3 +1,111 @@
+| *mplgs*                 | *None*   | short     | Number of lags estimated from the sequence                                              |
+| *mplgexs*               | *None*   | short     | Number of lags used for tauscan. This field originates from `fittex` algorithm and is not used in RAWACF file format. | 
+| *ifmode*                | *None*   | short     | A flag indicating that the receiver voltage samples were obtained by digitizing the signal at an intermediate frequency (and then digitally mixed to baseband) instead of mixing to baseband and then digitizing (`ifmode=0`).         |
+| *nrang*                 | *None*   | short     | Number of range range gates                                                             |
+| *frang*                 | *km*     | short     | Distance to the first range gate                                                        |
+| *rsep*                  | *km*     | short     | Range seperation                                                                        |
+| *xcf*                   | *None*   | short     | XCF flag, indicates if XCF for elevation angle estimation is available (1) or not (0)   |
+| *tfreq*                 | *kHz*    | short     | Transmitted frequency                                                                   |
+| *mxpwr*                 | *dB*     | int       | Maximum power                                                                           |
+| *lvmax*                 | *None*   | int       | Maximum noise level allowed                                                             |
+| *combf*                 | *None*   | string    | Control program name or command line, or comment buffer                                 |
+| *rawacf.revision.major* | *None*   | int       | Major version number of the RAWACF algorithm                                            |
+| *rawacf.revision.minor* | *None*   | int       | Minor version number of the RAWACF algorithm                                            |
+| *thr*                   | *None*   | float     | Threshold factor                                                                        |
+
+
+### Vectors 
+
+!!! Note
+    *slist* contains the range gates that obtained data points during the integration period of the beam. 
+
+| Field name  | Units    | Dimensionality | Data Type   | Description                                                                 |
+| :---------- | :-----:  | :-------:      | :---:       | :---                                                                        |
+| *ptab*      | *None*   | *[mppul]*      | short       | Pulse table                                                                 |
+| *ltab*      | *None*   | *[mplgs+1][2]* | short       | Lag table, showing which pulses to use to estimate each lag                 |
+| *pwr0*      | *dB*     | *[nrang]*      | float       | Lag zero power (actually SNR), estimated from voltage samples (not fitted)  |
```

Comment:
In rawacf files, is this in dB? I don't remember right now, just want to comment for a double check. If in units of dB and SNR, which noise is used? It might be useful to document this.

Reply:
Good catch @asreimer 
For rawacf files, pwr0 has arbitrary units, not dB. 
For fitacf files, pwr0 is in dB, where the reference is the sky noise.

Reply:
``` suggestion
| *pwr0*      | *None*     | *[nrang]*      | float       | Lag zero power (actually SNR), estimated from voltage samples (not fitted)  |
```
Would this be the change @ecbland ?

Reply:
@mts299  We also need to update the description since it's not a SNR at this stage.
```suggestion
| *pwr0*      | *None*     | *[nrang]*      | float       | Lag zero power, estimated from voltage samples (not fitted)  |
```